src/commands/ledger/download.ts

```diff
@@ -1,10 +1,10 @@
-import {Flags} from "@oclif/core"
-import {downloadAndValidateLedger} from "../../lib/download"
-import {TokenAwareCommand} from "../../lib/token-command"
-import { initClient } from '../../lib/client';
+import {Flags} from '@oclif/core'
```

Comment:
' vs " - my take, I use ' for JS files and " for TS as it reminds me of Java (type systems). I am half-seriously thinking of switching to ` for everything, but haven't made that change.